src/viz/viewer.cpp

```diff
@@ -387,24 +389,33 @@ static ViewerCam initCam(math::Vector3 pos, math::Quat rot)
     return cam;
 }
 
-Viewer::Impl::Impl(const Config &cfg)
+Viewer::Impl::Impl(const Config& cfg, const VoxelConfig& vox_cfg)
     : cam(initCam(cfg.cameraPosition, cfg.cameraRotation)),
-      frameCfg {
```

Comment:
Obviously there are numerous style issues. I recommend reading through the entire diff on this page or with git diff and checking each line.